### DISCUSSION TITLE
v0.48.1-20240928.193147

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#1087] [General]
* Bugfix: `lcd/cwd` fix.
* Dependencies updates:
  - sparkle-project/Sparkle@2.7.4
  - Quick/Nimble@13.4.0
  - apple/swift-collections@1.1.2

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.48.1-20240928.193147'>v0.48.1-20240928.193147</a>.</em>

**Comment by jasonm** [comment]
@qvacua Thank you for your work on VimR! I enjoy using it daily.
